addons/l10n_co/data/l10n_co_chart_data.xml

```diff
@@ -9,7 +9,7 @@
         <field name="cash_account_code_prefix">1105</field>
         <field name="transfer_account_code_prefix">1115</field>
         <field name="country_id" ref="base.co"/>
-        <field name="spoken_languages" eval="'en_US;es_CO'"/>
+        <field name="spoken_languages" eval="'en_US;es_CO;es_ES'"/>
```

Comment:
maybe this change is not necessary.

Reply:
es_ES seems to be the generic Spanish (also Spanish from Spain). I think this option impacts what languages are loaded during module installation, so could still be necessary?